prompts/codex/burlaki-human.md

```
@@ -1,28 +1,90 @@
+   3. If not found: exit 2 with `Story not found: <id>`.
+   4. If story.status != `blocked`: exit 2 with `Story <id> is not blocked (status: <status>)`.
+   5. Create checkpoint:
+      `git add -A && git commit -m "[burlaki-checkpoint] Resolved escalation: <story-id>"`
+   6. Update PRD:
+      - Set `status = "pending"`
+      - Add `resolution_note` if provided
+      - Clear `blocked_reason`
+   7. Atomic write PRD:
+      `jq '...' .agents/tasks/prd.json > .agents/tasks/prd.json.tmp && mv .agents/tasks/prd.json.tmp .agents/tasks/prd.json`
+   8. Output (bordered box format):
+      ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)